tools/e2etesting/pipeline.yml

```diff
@@ -53,7 +53,7 @@ stages:
     steps:
     - template: steps/runtests.yml
       parameters:
-        RunApiTests: true
```

Comment:
On a side note, shall we do some renaming of pipeline YAML files to be more explicit ?
* `pipeline.yml` -> `pipeline_orchestrated.yml` 
* `pipelinestandalone.yml` -> `pipeline_standalone.yml`

Reply:
Good Idea, we should do it in another PR